django/contrib/admin/options.py

```diff
@@ -973,10 +975,10 @@ def action_checkbox(self, obj):
     def _get_action_description(func, name):
         return getattr(func, "short_description", capfirst(name.replace("_", " ")))
 
-    def _get_base_actions(self):
+    def _get_base_actions(self, klass_actions):
```

Comment:
To avoid duplicate code, I re-write this method to accept the `actions` or `change_actions` by param. Probably there is a better way to implement this.